File: A1/A1-024/A1-024.py
# --------------------------------------------------
# File Name : A1-024.py
# Problem   : Car Tax
# Author    : Worralop Srichainont
# Date      : 2026-02-08
# --------------------------------------------------

# List of car tax
CAR_TAX = [[1250, 1400, 2000], [1100, 1300, 1700], [1000, 1200, 1500]]

# Input
year = int(input())
motor_volume = int(input())

# Find index for year
year_idx = -1
if year <= 1990:
    year_idx = 0
elif year <= 2000:
    year_idx = 1
else:
    year_idx = 2

# Find index for motor
motor_idx = -1
if motor_volume <= 1500:
    motor_idx = 0
elif motor_volume <= 2000:
    motor_idx = 1
else:
    motor_idx = 2

# Output tax
print(CAR_TAX[year_idx][motor_idx])

File: A1/A1-024/A1-024.cpp
#include<iostream>
using namespace std;

int main()
{
    // This makes std::cin and std::cout faster
    ios_base::sync_with_stdio(false); cin.tie(0);

    // Input
    int year, motor;
    cin >> year >> motor;
    // Output
    int TAX[3][3] = {{1250, 1400, 2000}, 
                     {1100, 1300, 1700},
                     {1000, 1200, 1500}};
    int YEAR_TYPE = 1;
    if(year <= 1990) { YEAR_TYPE = 0; }
    if(year >= 2000) { YEAR_TYPE = 2; }
    
    int MOTOR_TYPE = 1;
    if(motor <= 1500) { MOTOR_TYPE = 0; }
    if(motor > 2000) { MOTOR_TYPE = 2; }

    cout << TAX[YEAR_TYPE][MOTOR_TYPE];
    return 0;
}